appinventor/components/src/com/google/appinventor/components/common/YaVersion.java

```diff
@@ -514,7 +514,7 @@ private YaVersion() {
   // - BLOCKS_LANGUAGE_VERSION was incremented to 32
 
   // TODO: Bump this before merge. Include notes about all upgraded components.
-  public static final int YOUNG_ANDROID_VERSION = 207;
+  public static final int YOUNG_ANDROID_VERSION = 208;
```

Comment:
Did you just bump this to test? We normally don't increase it until we actually merge the whole change.